src/libs/actions/Task.js

```diff
@@ -309,8 +309,11 @@ function editTaskAndNavigate(report, ownerEmail, ownerAccountID, {title, descrip
     let optimisticAssigneeAddComment;
     let assigneeChatReportID;
     if (assigneeAccountID && assigneeAccountID !== report.managerID) {
```

Comment:
Can we just check here that the assigneeAccountID is also not equal to our own accountID and then you don't have to change the below?

Reply:
Hmm that means I need to supply the `currentUserAccountID` to `editTaskAndNavigate` right? Like we do in `createTaskAndNavigate`, or subscribe to onyx in the same file?

Reply:
Yeah that sounds good.